src/test/java/org/broadinstitute/hellbender/cmdline/GATKPlugin/GATKReadFilterPluginDescriptorTest.java

```diff
@@ -538,6 +538,27 @@ public void testEnableDisableConflict() {
                 "--" + ReadFilterArgumentDefinitions.DISABLE_READ_FILTER_LONG_NAME, "GoodCigarReadFilter"});
     }
 
+    @Test(expectedExceptions = CommandLineException.class)
```

Comment:
I'm not 100% sure, but I think there are issues with inverted and disabled read filters.  You should be allowed to 
a) disable the tool defaults and add back one of them but inverted.  
b) disable a specific read filter and add it back inverted

Could you add tests for those cases?

Reply:
I will add a test for A, B is very thorny... i'm going to say thats an edge case and not handle it as it is a whole lot more complicated for a very niche case (if you want to invert a default filter you must construct your own filter list)